app.py

```diff
@@ -1,12 +1,21 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, render_template, request, redirect, url_for, flash
 import session_items as session
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
-
+print ("server starting")
 @app.route('/')
 def index():
-    return 'Hello World!'
+    print ("handling request")
+    Todos = session.get_items()
```

Comment:
A minor point - you should keep to lower case snake_case for variable names.
Capitalised first letter usually indicates it's a class name instead. 

It doesn't change the functionality but naming conventions make it easier to read code.